### PR TITLE
Feature/improve file inputs

### PR DESCRIPTION
**Pull request by bauerfyr**
Checks that the file in the RFU file input is an RFU (based only on the naming convention) and does the same for the INFO file and input. Color coded indicators and a message inform the user of a correct or incorrect selection. 

The file input is told to accept .xls and .xlsx files only. The file input dialogue in Brave adheres to this, but Firefox unfortunately ignores it.

Improvements in the messages could be made, and further checks (such as matching dates, if this becomes useful in the future)

More styling will be added when the page is styled on another branch.

### CODE REVIEW

File: flaskr/static/js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        
+        if (name.endsWith(expected)) {
+            remClass(parent, "invalidinput");
+            addClass(parent, "validinput");
+        } else {
+            remClass(parent, "validinput");
+            addClass(parent, "invalidinput");
+        }
+    } else {
+        remClass(parent, ["validinput", "invalidinput"]);
```
Comment (seiboldfyr) [inline flaskr/static/js/scripts.js]: Here you handle if no files are added, right? 
Half of the ImportValidator execute function will now be redundant, we may not need it at all if/when you add a validation to check if a pre-existing dataset is selected.

File: flaskr/static/css/main.css [hunk 2]
```diff
@@ -51,4 +51,17 @@ h3 {
...
+    background-color: limegreen;
+}
+
+.invalidinput {
+    background-color: orange;
+}
+
+.invalidinput::after {
+    content: "⚠️ Double check that this is the requested file type!";
+    display: block;
```
Comment (seiboldfyr) [inline flaskr/static/css/main.css]: Great additions!

File: flaskr/templates/home.html [hunk 3]
```diff
@@ -40,6 +40,8 @@
 
 {% block content %}{% endblock %}
 
-{% block buttons %}{% endblock %}
+{% block buttons %}{% endblock %}function validateFile(fileType) {
+    console.log(this.value);
```
[the remark this answers is not in the document]
Comment (bauerfyr) [reply flaskr/templates/home.html]: Yep! I'll remove that!

File: flaskr/static/js/scripts.js (hunk 1 above)
Comment (bauerfyr) [reply flaskr/static/js/scripts.js]: I didn't add checks for "no file selected" selected here, but we could add a "Waiting for selection..." message.
Currently a white background will signal to users that there has been no selection. At least after they notice and are used to the change. This is just adding a css (for message and color) indication of whether a good file has been selected or we think something is wrong. We could add some more specific messages if we think of some useful ones.

Also it does not currently prevent the user from clicking through if they don't notice the bright indicator. It's more of a warning, although we could change that... Should we do it?  I didn't want to force a naming scheme, but it seems like they don't change so maybe it won't be a problem

Comment (seiboldfyr) [reply flaskr/static/js/scripts.js]: Warning is good, the validator will deliver an error message if they click to go further with a non-excel file.

**Pull request merged**